[v3] Blade component

Elhebert opened the pull request:
I finally found some time to draft a PR for the blade component.

For more details check #57

It's a first draft, but here's what the API could look like. This would simplify the usage of the package.

```html
<x:sri.script src="/public/js/app.js" />
<!-- <script src="{{ asset("/public/js/app.js") }}" integrity= "..." /> -->

<x:sri.script.mix src="/public/js/app.js />
<!-- <script src="{{ mix("/public/js/app.js") }}"  integrity= "..." /> -->

<x:sri.link href="/public/css/app.css" rel="stylesheet" />
<!-- <link href="{{ asset("/public/css/app.css") }} rel="stylesheet"  integrity= "..." /> -->

<x:sri.link.mix href="/public/css/app.css" rel="stylesheet" />
<!-- <link href="{{ mix("/public/css/app.css") }} rel="stylesheet"  integrity= "..." /> -->
```

juliomotol commented on src/SriServiceProvider.php:
```diff
@@ -27,12 +29,9 @@ public function boot()
             __DIR__.'/../config/subresource-integrity.php' => config_path('subresource-integrity.php'),
         ]);
 
-        Blade::directive('mixSri', function ($arguments) {
-            return "<?php echo app('".Sri::class."')->mix({$arguments}) ?>";
-        });
-
-        Blade::directive('assetSri', function ($arguments) {
-            return "<?php echo app('".Sri::class."')->asset({$arguments}) ?>";
-        });
```
Do we no longer want to support Laravel 6.x (which is still in LTS)? The view class components only came in v7.x.

Elhebert replied:
v3 will drop the support for older Laravel version.

Since Laravel 8 was just released, I'm still undecided to drop Laravel 7 as well 🤔

juliomotol replied:
Hmmm... I see. All good then.

Elhebert commented:
Just added basic tests and cleaned up after removing the blade directive. 
I also updated the API a little bit and moved the `mix` part as an attribute to you have the same component for `asset` and `mix` which is less maintenance 🤔

```html
<x:sri.script src="/public/js/app.js" />
<!-- <script src="{{ asset("/public/js/app.js") }}" integrity= "..." /> -->

<x:sri.script src="/public/js/app.js" mix />
<!-- <script src="{{ mix("/public/js/app.js") }}" integrity= "..." /> -->

<x:sri.link href="/public/css/app.css" rel="stylesheet" />
<!-- <link href="{{ asset("/public/css/app.css") }} rel="stylesheet"  integrity= "..." /> -->

<x:sri.link href="/public/css/app.css" rel="stylesheet" mix />
<!-- <link href="{{ mix("/public/css/app.css") }} rel="stylesheet"  integrity= "..." /> -->
```

Elhebert commented:
I decided to drop Laravel 7 support. I had to many issues with the tests.

I also added a `@once` directive around the component to never re-render the component more than once.
